### ISSUE TITLE
conda distribution not up to date

### ISSUE THREAD

**Issue by cpaton8** [conda-forge/rasterio-feedstock#192]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:
I was wondering if there is a regular schedule on which the conda version is pulled from source? There was a change to rasterio main in the last month that is not currently on the conda install version and it is extremely difficult to install rasterio from source on a Windows computer. 

If curious, the change at question is the ability to resample using sum method which is updated in enums.py in the Resampling class. If downloaded via conda, sum=13 is missing for this class. 

Thank you!

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

# Name                    Version                   Build  Channel
affine                    2.3.0                      py_0    conda-forge
attrs                     20.3.0             pyhd3deb0d_0    conda-forge
boost-cpp                 1.74.0               h54f0996_2    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2020.12.5            h5b45459_0    conda-forge
cairo                     1.16.0            hb19e0ff_1008    conda-forge
certifi                   2020.12.5        py37h03978a9_1    conda-forge
cfitsio                   3.470                h0af3d06_7    conda-forge
click                     7.1.2              pyh9f0ad1d_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.7.1              pyhd8ed1ab_0    conda-forge
curl                      7.75.0               hf1763fc_0    conda-forge
expat                     2.2.10               h39d44d4_0    conda-forge
fontconfig                2.13.1            h1989441_1004    conda-forge
freetype                  2.10.4               h546665d_1    conda-forge
freexl                    1.0.6                ha8e266a_0    conda-forge
gdal                      3.1.4           py37haf026f5_10    conda-forge
geos                      3.9.1                h39d44d4_2    conda-forge
geotiff                   1.6.0                h8e90983_5    conda-forge
gettext                   0.19.8.1          h1a89ca6_1005    conda-forge
hdf4                      4.2.13            h0e5069d_1004    conda-forge
hdf5                      1.10.6          nompi_h5268f04_1114    conda-forge
icu                       68.1                 h0e60522_0    conda-forge
intel-openmp              2020.3             h57928b3_311    conda-forge
jpeg                      9d                   h8ffe710_0    conda-forge
kealib                    1.4.14               h96bfa42_2    conda-forge
krb5                      1.17.2               hbae68bd_0    conda-forge
libblas                   3.9.0                     8_mkl    conda-forge
libcblas                  3.9.0                     8_mkl    conda-forge
libcurl                   7.75.0               hf1763fc_0    conda-forge
libffi                    3.3                  h0e60522_2    conda-forge
libgdal                   3.1.4               hbe61683_10    conda-forge
libglib                   2.68.0               h1e62bf3_2    conda-forge
libiconv                  1.16                 he774522_0    conda-forge
libkml                    1.3.0             h02ac0ef_1012    conda-forge
liblapack                 3.9.0                     8_mkl    conda-forge
libnetcdf                 4.7.4           nompi_h3a9aa94_107    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libpq                     13.1                 h4f54205_2    conda-forge
librttopo                 1.1.0                hb340de5_6    conda-forge
libspatialite             5.0.1                h6b539a6_4    conda-forge
libssh2                   1.9.0                h680486a_6    conda-forge
libtiff                   4.2.0                hc10be44_0    conda-forge
libwebp-base              1.2.0                h8ffe710_2    conda-forge
libxml2                   2.9.10               hf5bbc77_3    conda-forge
lz4-c                     1.9.3                h8ffe710_0    conda-forge
mkl                       2020.4             hb70f87d_311    conda-forge
numpy                     1.20.1           py37hd20adf4_0    conda-forge
openjpeg                  2.4.0                h48faf41_0    conda-forge
openssl                   1.1.1j               h8ffe710_0    conda-forge
pcre                      8.44                 ha925a31_0    conda-forge
pip                       21.0.1             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               h8ffe710_0    conda-forge
poppler                   21.03.0              h9ff6ed8_0    conda-forge
poppler-data              0.4.10                        0    conda-forge
postgresql                13.1                 h0f1a9bc_2    conda-forge
proj                      8.0.0                h1cfcee9_0    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
python                    3.7.10          h7840368_100_cpython    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
rasterio                  1.2.1            py37hc4b0cd6_0    conda-forge
setuptools                49.6.0           py37h03978a9_3    conda-forge
snuggs                    1.4.7                      py_0    conda-forge
sqlite                    3.35.2               h8ffe710_0    conda-forge
tiledb                    2.2.6                hf84e3da_0    conda-forge
tk                        8.6.10               h8ffe710_1    conda-forge
vc                        14.2                 hb210afc_4    conda-forge
vs2015_runtime            14.28.29325          h5e1d092_4    conda-forge
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
wincertstore              0.2             py37h03978a9_1006    conda-forge
xerces-c                  3.2.3                h0e60522_2    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
zlib                      1.2.11            h62dcd97_1010    conda-forge
zstd                      1.4.9                h6255e5f_0    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : raster-pipeline
    active env location : C:\Users\cpato\Anaconda3\envs\raster-pipeline
            shell level : 2
       user config file : C:\Users\cpato\.condarc
 populated config files : C:\Users\cpato\.condarc
          conda version : 4.9.2
    conda-build version : 3.18.9
         python version : 3.7.4.final.0
       virtual packages : __win=0=0
                          __archspec=1=x86_64
       base environment : C:\Users\cpato\Anaconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\cpato\Anaconda3\pkgs
                          C:\Users\cpato\.conda\pkgs
                          C:\Users\cpato\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\cpato\Anaconda3\envs
                          C:\Users\cpato\.conda\envs
                          C:\Users\cpato\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.9.2 requests/2.25.0 CPython/3.7.4 Windows/10 Windows/10.0.18362
          administrator : False
             netrc file : None
           offline mode : False

```
</details>

**Comment by xylar** [comment]
@cpaton8, we just build releases of rasterio, and 1.2.1 (which you have in your environment) appears to be the latest release.  If you are requesting information on when the next release will occur, you need to do that "upstream" with the package develpers: https://github.com/mapbox/rasterio

**Comment by cpaton8** [comment]
Great thank you @xylar

**Comment by cpaton8** [comment]
Actually @xylar this fix was added to main in [February ](https://github.com/mapbox/rasterio/commit/4eaaf5d97e75189435400570a39cf4ade11ac98b#diff-c478f474cf2652754a1f7ad18c703bc793c9f0be700513f6fe103d40c728a7d0) and the current release was dated to March 3. Is this an issue I should flag upstream?

**Comment by xylar** [comment]
Yes, flag that upstream, please. We are hesitant to patch source unless packages are truly broken.

**Comment by xylar** [comment]
If you look in the code in the most recent release, you'll see that the change you're after isn't there yet:
https://github.com/mapbox/rasterio/blob/1.2.1/rasterio/enums.py

**Comment by xylar** [comment]
I believe the reason is that the PR got merged into `master`, not `maint-1.2`.  But I am not at all familiar with rasterio's development practices.

**Comment by cpaton8** [comment]
Thanks for the help @xylar, I flagged upstream which can be seen here: https://github.com/mapbox/rasterio/issues/2137.
